workshop/selenium.rb

```diff
@@ -0,0 +1,109 @@
+
+require 'selenium-webdriver'
+require 'uri'
+
+module SeleniumFramework
+
+    $driver
+    $wait
+
+    def start(url)
+        begin
+            options = Selenium::WebDriver::Chrome::Options.new
+            options.add_argument('--headless')
+            options.add_argument("--no-sandbox");
+            options.add_argument("--disable-dev-shm-usage");
+            Selenium::WebDriver::Chrome::Service.driver_path = "/usr/local/bin/chromedriver"
+            $driver = Selenium::WebDriver.for :chrome ,  options: options
```

Comment:
Refactor the driver code into another class/module and then pass the driver object as an argument to this method